cli/src/commands/apps/updateApp.ts

```diff
@@ -0,0 +1,57 @@
+import { flags } from '@oclif/command'
+import chalk from 'chalk'
+import { IAppMetadata, IMemberRemarked, MemberRemarked } from '@joystream/metadata-protobuf'
+import { metadataToBytes } from '../../helpers/serialization'
+import { getInputJson } from '../../helpers/InputOutput'
+import AppCommandBase from '../../base/AppCommandBase'
+import MembershipsCommandBase from '../../base/MembershipsCommandBase'
+
+export default class GenerateAppCreationMessage extends AppCommandBase {
+  static description = 'App creation message factory'
+
+  static flags = {
+    input: flags.string({
+      required: false,
+      char: 'i',
+      description: `Path to JSON file containing app details`,
+    }),
+    appId: flags.string({
+      required: true,
+      description: `ID of the app to update`,
+    }),
+    skip: flags.boolean({
+      char: 's',
+      description: "If true command won't prompt missing fields",
+    }),
+    ...MembershipsCommandBase.flags,
+  }
+
+  async run(): Promise<void> {
+    let updateAppRemarked: IMemberRemarked | null
+    const { input, appId, skip } = this.parse(GenerateAppCreationMessage).flags
+    if (input) {
+      const inputBody = await getInputJson<IAppMetadata>(input)
+      const appMetadata = skip ? inputBody : await this.promptAppMetadata(inputBody)
```

Comment:
Unless using a combination of `--input` and `--skip` it's impossible to partially update the app.
I suggest not setting any value of `prev[key]` (or setting it to `null` / `undefined`) if `promptValue` is an empty string in `promptAppMetadata`.
This of course introduces another issue, which is inability to explicity _unset_ a filed, but I think that's ok (unless you want to solve that one too, for exmaple, by asking whether user wants to update each given field or not)

Reply:
Maybe just pressing ENTER to skip or pass a single dash to unset a field will be straightforward enough. Can't think of any misusage.

Looks like metaprotocol drops fields with a `null` value, and the same goes for `undefined`, to resolve it I went with an empty string to unset a field.